packages/web/src/utils.test.ts

```diff
@@ -0,0 +1,62 @@
+import { parseProperties } from './utils';
+
+describe('utils', () => {
+  describe('parse properties', () => {
+    describe('strip', () => {
+      it('should allow all properties', () => {
+        const properties = {
+          number: 10,
+          string: 'some-string',
+          boolean: true,
+          nullable: null,
+        };
+
+        const parsed = parseProperties(properties, { strip: true });
+
+        expect(properties).toEqual(parsed);
+      });
+
+      it('should dismiss array and object', () => {
+        const properties = {
+          string: 'some-string',
+          array: [],
+          object: {},
+        };
+
+        const parsed = parseProperties(properties, { strip: true });
+
+        expect({
+          string: 'some-string',
+        }).toEqual(parsed);
+      });
+    });
+
+    describe('throw error', () => {
+      it('should allow all properties', () => {
+        const properties = {
+          number: 10,
+          string: 'some-string',
+          boolean: true,
+          nullable: null,
+        };
+        const parsed = parseProperties(properties, {
+          strip: false,
+        });
+
+        expect(properties).toEqual(parsed);
+      });
```

Comment:
Should this be in a different `describe` as it's not throwing?

Reply:
it tests the case where it would throw (and not strip) in case there is an error. but its not which is fine!

Reply:
I only tested `test` once in `generic.test.ts` as the react export is the same function exported.